test.js

```diff
@@ -19,6 +19,7 @@ describe('updateNotifier', () => {
 			callback: options.callback || null
 		};
 	};
+	const argv = process.argv.slice(0);
```

Comment:
Yeah I thought the same thing at first (and was really confused when code coverage dropped after adding your tests), but unfortunately doing `process.argv = argv` means we have to get a new shallow copy each time to avoid side effects of `process.argv.push(/*...*/)` in subsequent tests (since both variables reference the same array object).